internal/rbd/rbd_util.go

```diff
@@ -148,6 +146,8 @@ type rbdImage struct {
 
 	// Set metadata on volume
 	EnableMetadata bool
+	// ParentInTrash indicates the parent image is in trash.
+	ParentInTrash bool
```

Comment:
why move it around? grouping all parent related attributes looks nicer to me

Reply:
> why move it around? grouping all parent related attributes looks nicer to me

A linter throws up error when i do that and expects grouping similar data types together for better padding in the struct.